src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/DraftApprenticeshipControllerTests/AddDraftApprenticeshipTests.cs

```diff
@@ -128,8 +128,6 @@ public AddDraftApprenticeshipTestsFixture()
             AuthorizationService = new Mock<IAuthorizationService>();
```

Comment:
this can be removed as the controller no longer requires the dependency

Reply:
this has been removed